Feature/rm dynamic bld Remove dynamic build from Tech Doc (#22)
NCAR/ccpp-doc: CCPPtechnical/source/Glossary.rst

      two or more schemes together, or provide code for conversions, initializing sums, or
      applying tendencies, for example.

   Multi-suite CCPP build

>       A CCPP build type in which a set of physics suites is specified

Reviewer: Given that there is only one CCPP build type, it seems strange to refer to "a CCPP build type".
```suggestion
      CCPP build property in which a set of physics suites is specified
```

      oversimplified parameters, rather than realistically requiring such effects to be 
      consequences of the dynamics of the system (AMS Glossary)

   Physics *cap*
      Autogenerated interface between an individual physics scheme and the host model. Used
      only in the dynamic CCPP build

> 

Reviewer: I would like to retain an entry for physics cap because this terminology is still used in places in this documentation and in the code. Suggested definition: Generic name to refer to suite and group physics caps.

>    Suite *cap*

Reviewer: ```suggestion
   Physics Suite *cap*
```
Reorder so it is in the correct alphabetical location.

Also, please change "Group cap" entry to "Physics group cap".

      Autogenerated interface between an entire suite of physics schemes and the host model.

>       They consist of calls to autogenerated group caps. They may be used to call an entire suite
>       at once or to call a specific group within a physics suite

Reviewer: Singular.
```suggestion
      It consists of calls to autogenerated physics group caps. It may be used to call an entire suite
      at once or to call a specific group within a physics suite
```
